content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,127 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+feature:
+  title: Topology Aware Hints
+  description: >
+    Enables topology aware routing with topology hints on EndpointSlices.
+
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in EndpointSlices. This approach tries to keep traffic close to where
+it originated from. This can result in decreased costs and improved performance.
+
+<!-- body -->
+
+## Introduction
+
+Kubernetes clusters are increasingly deployed in multi-zone environments.
+_Topology Aware Hints_ adds new functionality that can help keep traffic within
```

Comment:
nit: in a concept page, we document the feature as if you're using it. That's different from a release note or a
release blog article where we highlight that the feature is new.

How about:
```suggestion
 _Topology Aware Hints_ provide a mechanism to help keep traffic within
```

Reply:
Updated, thanks!